resources/EPvalidator.rb

```diff
@@ -410,6 +410,8 @@ def self.run_validator(hpxml_doc)
         "[WaterHeaterType='storage water heater' or WaterHeaterType='instantaneous water heater' or WaterHeaterType='heat pump water heater' or WaterHeaterType='space-heating boiler with storage tank' or WaterHeaterType='space-heating boiler with tankless coil']" => one, # See [WHType=Tank] or [WHType=Tankless] or [WHType=HeatPump] or [WHType=Indirect] or [WHType=CombiTankless]
         "[Location='living space' or Location='basement - unconditioned' or Location='basement - conditioned' or Location='attic - unvented' or Location='attic - vented' or Location='garage' or Location='crawlspace - unvented' or Location='crawlspace - vented']" => one,
         "FractionDHWLoadServed" => one,
+        "[EnergyFactor | UniformEnergyFactor]" => one,
+        "WaterHeaterInsulation/Jacket/JacketRValue" => zero_or_one, # Capable to model tank wrap insulation
```

Comment:
We will need to decide which water heating system types to allow this for. For example, we probably shouldn't offer it as an input for tankless or tankless coil, so the above specification should probably become conditional for certain water heater types. Ideally we'd allow it for conventional storage, HPWHs, and indirect water heaters.

Reply:
Do we want to add error checks for those combinations not allowed (tankless + jacket)? Since it seems even if we remove this line to the certain water heater type blocks, it didn't give proper errors but simply ignored this input, correct?

Reply:
That's correct, it would be ignored. If someone accidentally provided a jacket insulation R-value for a tankless system, it shouldn't affect results, so ignoring it seems fine to me. (Or maybe they'd want to specify a jacket R-value of zero, to simplify their HPXML export code across different system types?)

Reply:
Emm I am not sure if I followed. I didn't read this input in the tankless tank types in measure.rb for apply_tankless function and inside the `` calc_tank_ua`` function, it also bypassed the tankless ua recalculation for jacket. So it would ignore the input for those systems even if the EPvalicator checked that (zero_or_one might be fine? ) But removing this line to be inside of different types would be more clear to see the available combinations.